src/encoded/dev_servers.py

```diff
@@ -1,165 +1,3 @@
-    # Loading app will have configured from config file. Reconfigure here:
-    logging.getLogger('encoded').setLevel(logging.INFO)
-
-    # get the config and see if we want to connect to non-local servers
-    # config = get_appsettings(args.config_uri, args.app_name)
-
-    datadir = os.path.abspath(args.datadir)
-    pgdata = os.path.join(datadir, 'pgdata')
-    esdata = os.path.join(datadir, 'esdata')
-    ### comment out from HERE...
-    if args.clear:
-        for dirname in [pgdata, esdata]:
-            if os.path.exists(dirname):
-                shutil.rmtree(dirname)
-    if args.init:
-        postgresql_fixture.initdb(pgdata, echo=True)
-    ### ... to HERE to disable recreation of test db
-    ### may have to `rm /tmp/snovault/pgdata/postmaster.pid`
-
-    postgres = postgresql_fixture.server_process(pgdata, echo=True)
-    elasticsearch = elasticsearch_fixture.server_process(esdata, echo=True)
-    nginx = nginx_server_process(echo=True)
-    processes = [postgres, elasticsearch, nginx]
-
-
-    @atexit.register
-    def cleanup_process():
-        for process in processes:
-            if process.poll() is None:
-                process.terminate()
-        for process in processes:
-            try:
-                for line in process.stdout:
-                    sys.stdout.write(line.decode('utf-8'))
-            except IOError:
-                pass
-            process.wait()
-
-    if not args.no_ingest:
-        ingestion_listener = ingestion_listener_process(args.config_uri, args.app_name)
-        processes.append(ingestion_listener)
-
-    app = get_app(args.config_uri, args.app_name)
-
-    # clear queues and initialize indices before loading data. No indexing yet.
-    # this is needed for items with properties stored in ES
-    if args.init:
-        create_mapping.run(app, skip_indexing=True, purge_queue=True)
-
-    if args.init and args.load:
-        load_test_data = app.registry.settings.get('load_test_data')
-        load_test_data = DottedNameResolver().resolve(load_test_data)
-        load_res = load_test_data(app)
-        if load_res:  # None if successful
-            raise load_res
-
-        # now clear the queues and queue items for indexing
-        create_mapping.run(app, check_first=True, strict=True, purge_queue=True)
-
-    PRINT('Started. ^C to exit.')
-
-    stdouts = [p.stdout for p in processes]
-
-    # Ugly should probably use threads instead
-    while True:
-        readable, writable, err = select.select(stdouts, [], stdouts, 5)
-        for stdout in readable:
-            for line in iter(stdout.readline, b''):
-                sys.stdout.write(line.decode('utf-8'))
-        if err:
-            for stdout in err:
-                for line in iter(stdout.readline, b''):
-                    sys.stdout.write(line.decode('utf-8'))
-            break
-
-
-if __name__ == '__main__':
-    main()
+from snovault import dev_servers
+from . import project_defs
+dev_servers.main()
```

Comment:
This is going to cause problems to anyone just loading this module. I think you want (untested):
```suggestion
if __name__ == '__main__':
    dev_servers.main()
```